tests/test-core.js

```diff
@@ -322,11 +322,11 @@ I have a question. Can you help answer it? –[[User:Novem Linguae|<span style="
 		const revId = 592496;
 		const isBiography = true;
 		const newWikiProjects = [ 'WikiProject Africa', 'WikiProject Alabama' ];
-		const lifeStatus = 'living';
+		const lifeStatus = 'blp';
```

Comment:
Internally in AFCH (but not in the wikicode), can we keep using "living" as the value? I find living to be more readable.